### DISCUSSION TITLE
SER + RE模型串联测试的时候无法执行成功，报两种错误

### DISCUSSION THREAD

**Discussion by qdhonker** [PaddlePaddle/PaddleOCR#13368] [Q&A] [answered]
我在测试SER + RE模型串联任务的时候
使用了两条命令
1.python ./tools/infer_kie_token_ser_re.py -c configs/kie/vi_layoutxlm/re_test.yml -o Architecture.Backbone.checkpoints=./pretrained_model/re_vi_layoutxlm_xfund_pretrained/best_accuracy Global.infer_img=./ppocr_img/danju/3.jpg Global.infer_mode=False -c_ser configs/kie/vi_layoutxlm/test.yml -o_ser Architecture.Backbone.checkpoints=./pretrained_model/ser_vi_layoutxlm_xfund_pretrained/best_accuracy

2.python ./tools/infer_kie_token_ser_re.py -c configs/kie/vi_layoutxlm/re_test.yml -o Architecture.Backbone.checkpoints=./pretrained_model/re_vi_layoutxlm_xfund_pretrained/best_accuracy Global.infer_img=./ppocr_img/danju/3.jpg -c_ser configs/kie/vi_layoutxlm/test.yml -o_ser Architecture.Backbone.checkpoints=./pretrained_model/ser_vi_layoutxlm_xfund_pretrained/best_accuracy

第一条命令报错反馈如下：
![2C77CC79-F73B-4d7c-AD6B-9F5090BCDCE7](https://github.com/user-attachments/assets/6e12e5ae-1011-43c9-9240-47740a728ab7)

第二条命令摄氏反馈如下：
![FF48A27E-C6A2-4adf-9488-9D16513511F8](https://github.com/user-attachments/assets/f1d9f9cd-e567-4265-af75-744214233377)

**Comment by UserWangZz** [comment]
尝试使用paddlepaddle=2.4.2， paddlenlp=2.5.2版本推理试试

**Reply by UserWangZz** [reply]
这个错误是因为paddlenlp版本升级导致的

**Reply by qdhonker** [reply]
cuda和cuDNN对应的版本是什么，我试了11.2还是不行，
import paddle
paddle.utils.run_check()
检测没办法通过

**Reply by UserWangZz** [reply, marked as answer]
不好意思，paddlepaddle版本给错了，我这边是
![image](https://github.com/user-attachments/assets/21329974-6273-40f1-b4e4-7dc670a541a3)
测试是可以推理成功的

**Reply by qdhonker** [reply]
能跑成功了，感谢大佬
